tests/suites/coslite/cl.sh

```diff
@@ -0,0 +1,57 @@
+# Ensure that COS Lite deploys successfully, the
+# relations are setup as expected and we can access the dashboards.
+run_deploy_coslite() {
+	echo
+
+	local model_name file overlay_path admin_passwd alertmanager_ip grafana_ip prometheus_ip
+	model_name="deploy-coslite"
+	file="${TEST_DIR}/${model_name}.log"
+	ensure "${model_name}" "${file}"
+
+	overlay_path="./tests/suites/coslite/overlay"
+	juju deploy cos-lite --trust --channel=edge --overlay "${overlay_path}/offers-overlay.yaml" --overlay "${overlay_path}/storage-small-overlay.yaml"
+
+	echo "Wait for all coslite bundle charms to be active"
+	wait_for 5 '[.applications[] | select(."application-status".current == "active")] | length' 1800
```

Comment:
This seems a little fragile. I see no reason why cos-lite won't add more applications to the bundle in future. I would try and construct a statement around having no applications in non-active states. 

Perhaps you could add this as an `includes`. I know it will also be useful for magma and kubeflow

Reply:
Or just check for each of the applications as I was doing before? Actually I got this idea from magma itself.

Reply:
EDIT: Makes sense. We would expect no application to be non-active equating that to 0.

Reply:
It might be better to use `idle_condition` somehow, as I believe an application can still be active whilst it is doing things. I believe (although I know there are many exceptions) that's what we tend to do elsewhere

Reply:
This won't work for the applications stay in active state for a very very long time, I guess they tend to say that way, 
only the unit workloads tend to get idle. So am reverting it to the previous behavior of waiting explicitly for each of the charms in thebundle to attain the active state.

Reply:
EDIT: I think you might be correct. I got confused here sometimes the traefik stalls installing the agent giving me some mixed conclusions. Other times it is grafana take  a long time installing the agent.